src/parse-templates.ts

```diff
@@ -95,21 +68,14 @@ function isEscaped(template: string, _offset: number | undefined) {
  */
 export function parseTemplates(
   template: string,
-  relativePath: string,
-  options?: ParseTemplatesOptions
```

Comment:
I think we still probably want to accept options like before (basically the same signature as original), but update `TemplateLiteralMatch` to include more metadata:

```ts
export interface TemplateLiteralMatch {
  type: 'template-literal';
  tagName: string;
  importConfig: StaticImportConfig;
  start: RegExpMatchArray;
  end: RegExpMatchArray;
}
```